src/assets/scripts/client/InputController.js

```diff
@@ -713,6 +714,16 @@ export default class InputController {
     }
 
     /**
+     * Disable default behavior of right-click menu appearing on right click
+     *
+     * @for InputController
+     * @method _disableRightClickMenu
+     */
+    _disableRightClickMenu() {
```

Comment:
Internet seems to indicate this is how the right click menu has to be handled. The standard preventDefault didn't prevent the right click menu from appearing